content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md

```diff
@@ -37,6 +37,14 @@ Each CODEOWNERS file assigns the code owners for a single branch in the reposito
 
 For code owners to receive review requests, the CODEOWNERS file must be on the base branch of the pull request. For example, if you assign `@octocat` as the code owner for *.js* files on the `gh-pages` branch of your repository, `@octocat` will receive review requests when a pull request with changes to *.js* files is opened between the head branch and `gh-pages`.
 
+## CODEOWNERS and forks
+
+Asides being used to tell at a glance who is responsible for a given file, CODEOWNERS are also used for triggering review requests for a pull request that modifies an owned file.
+
+Pull requests use the version of CODEOWNERS from the base branch of a pull request. A base branch is the branch that a pull request will modify if the pull request is merged. If the repo that you forked from is the base of your PR, then the PR will use the CODEOWNERS file from the repo that you forked from. If your PR is targeting a branch within your forked repo, then the PR will use the CODEOWNERS on that base branch in your forked repo.
```

Comment:
@isaacmbrown 's suggestions are pretty good.

I'd note that while the text here says `owners`, it's possible they're teams instead of owners. That should be addressed (I'm not sure it is currently, but any new text should also keep that in mind).

And it isn't precisely the `same permissions` as it is `permissions that allow <whatever the minimum is for reviewing>` it might be `write` (I can't remember, but we tripped on this recently in a private repository where a user was a member of the org but their permissions were insufficient -- I could have made them an `admin` of the repository and that wouldn't be the "same" permission as other repositories, but it would still have worked).